tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Catalogue Solutions Create.feature

```diff
@@ -0,0 +1,80 @@
+﻿Feature: Create catalogue solution order item
+    As a Buyer User
+    I want to create a catalogue solution order item for a given order
+    So that I can complete my order
+
+Background:
+    Given Orders exist
+        | OrderId    | Description      | OrganisationId                       | LastUpdatedByName | LastUpdatedBy                        |
+        | C000014-01 | Some Description | 4af62b99-638c-4247-875e-965239cd0c48 | Tom Smith         | 335392e4-4bb1-413b-9de5-36a85c9c0422 |
+    And Service Recipients exist
+        | OdsCode | Name              | OrderId    |
+        | ODS1    | Service Recipient | C000014-01 |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+@7840
+Scenario: 1. Create catalogue solution order item
+    Given the user creates a request to add a new catalogue solution order item to the order with ID 'C000014-01'
+    And the user enters '<payload-type>' payload
+    When the user sends the create catalogue solution order item request
+    Then a response with status code 201 is returned
+    And the expected catalogue solution order item is created
+
+    Examples: Request payloads
```

Comment:
Is there anypoint doing an examples for just one item? Or are you expecting more?

Reply:
I am expecting more once tiered comes in & possibly @CallumBradburyBJSS validation task.